data/campaigns/Secrets_of_the_Ancients/utils/zombie-utils.cfg

```diff
@@ -13,81 +13,97 @@
-                You can now raise rat corpses!,
-                You can now raise wolf corpses!,
-                You can now raise human corpses!,
-                You can now raise mounted corpses!,
-                You can now raise wolf rider corpses!,
-                You can now raise swimming corpses!,
-                You can now raise saurian corpses!,
-                You can now raise wose corpses!,
-                You can now raise goblin corpses!,
-                You can now raise gryphon corpses!,
-                You can now raise spider corpses!,
-                You can now raise troll corpses!,
-                You can now raise dwarf corpses!" # po: Be careful not to delete any commas!
-                key=allow_recruit_message
-                separator=","
-            [/split]
-        [/set_variables]
-        [set_variables]
-            name=zombies
-            mode=merge
-            [split]
-                # The player starts out with no zombies available.
-                list="no,no,no,no,no,no,no,no,no,no,no,no,no,no"
-                key=allow_recruit
-                separator=","
-            [/split]
+            [value]
+                type=SotA Flying Corpse_Bat
+                sota_variation= _ "(Bat)"
+                allow_recruit_message= _ "You can now raise bat corpses!"
+                allow_recruit=no
+            [/value]
+            [value]
+                type=SotA Walking Corpse_Rat
+                sota_variation= _ "(Rat)"
+                allow_recruit_message= _ "You can now raise rat corpses!"
+                allow_recruit=no
+            [/value]
+            [value]
+                type=SotA Walking Corpse_Wolf
+                sota_variation= _ "(Wolf)"
+                allow_recruit_message= _ "You can now raise wolf corpses!"
+                allow_recruit=no
+            [/value]
+            [value]
+                type=SotA Walking Corpse_Human
+                sota_variation= _ "(Human)"
+                allow_recruit_message= _ "You can now raise human corpses!"
+                allow_recruit=no
+            [/value]
+            [value]
+                type=SotA Walking Corpse_Mounted
+                sota_variation= _ "(Mounted)"
+                allow_recruit_message= _ "You can now raise mounted corpses!"
+                allow_recruit=no
+            [/value]
+            [value]
+                type=SotA Walking Corpse_Wolf Rider
+                sota_variation= _ "(Wolf Rider)"
+                allow_recruit_message= _ "You can now raise wolf rider corpses!"
+                allow_recruit=no
+            [/value]
+            [value]
+                type=SotA Swimming Corpse
+                # po: The ^ at the end means that this string is not used in English. Since there is only one swimmer corpse, it would be redundant to label it in English. You may translate this string (without the ^) if it makes more sense in your language to do so.
```

Comment:
While what the text is saying makes sense to me, it doesn't say where this will be used, and I had to playtest (in debug mode, for speed) to find out how these strings fit in. I still don't understand why these are separate strings, instead of just being included in the `[unit_type]name=` string.

Having a string translated to an empty string is bad for the translation stats - it will look like an untranslated string, and that's what gettext.wesnoth.org and poedit will treat it as.

Suggestion, add something like this:
These sota_variation strings are shown in the recruit dialog, appended to the name of the unit type. For example, "Flying Corpse (Bat)", "Walking Corpse (Human)" and "Walking Corpse (Rat)". There is only one "Swimming Corpse".

Reply:
I think changing it to "(Merfolk)" would fit, both from translation and from UX perspective. The player is definitely going to kill some Merfolk in S10, and the first scenario with Nagas is S11.

Just noticed a plot hole: when Ras-Tabahn kills Merfolk in S10, Ardonna gets the knowledge before they meet up again. Haven't playtested to check that, I'm just assuming it from the mechanics.

Reply:
This isn't really related, but I think it'd be nice if merfolk and naga had separate walking corpse variations. After all, the merfolk are more fish-like while nagas are more snake-like - they're not all that similar.